### DISCUSSION TITLE
Contributions des partenaires externes

### DISCUSSION THREAD

**Discussion by houssamkhalifeh** [VilledeMontreal/MontrealEnCommun#3] [Ideas]
Bonjour Jérémie @psionikangel et tout le monde,

On aimerait avoir plus d’échanges avec nos partenaires sur des questions techniques. On se demande si on peut ajouter un repo GitHub où ces échanges sont documentés et accessibles. 
J’ai changé le nom du repo principes-mec à **MontrealEnCommun** pour qu’il soit notre espace de collaboration. Je créerais des sous répertoires pour catégoriser la documentation. 
https://github.com/VilledeMontreal/MontrealEnCommun 
Une petite question : Les invités seront-il capable de forker, ajouter leurs documents (.md) puis faire des _Pull Requests_? Ils ne sont pas des membres comme vous le savez.
Merci,

**Comment by OlivierAlbertini** [comment]
Salut Houssam,

Du moment que le repo est publique, les gens pourront cloner et faire des PR. Ça nécessitera néanmoins un CLA pour pouvoir accepter la PR comme stipulé dans notre politique du libre.

Pour plus de détails, je t'invite à venir nous parler sur Google chat "LIBRE - Demandes de nouvelles contributions/publications" ou venir sur les séances du comité du libre le jeudi de 14h à 15h.

Merci

**Comment by vince-fabmob** [comment]
Salut Houssam, pour Fabrique des mobilités, il faudrait changer le lien du Git https://github.com/FabmobQC @houssamkhalifeh

**Reply by vince-fabmob** [reply]
Oh! J'ai réussi à le changer moi même. Yes!

**Reply by houssamkhalifeh** [reply]
Oui, tu as le droit @vince-fabmob  comme maintainer.
C'est une page temporaire en attendant que Larbi @lgharib prend en charge le maintien de leurs répertoire. Il n'a pas encore accepté l'invitation pour que je lui assigne la tâche.

**Reply by vince-fabmob** [reply]
Pour Solon, est-ce possible de mettre le dossier? Je pourrai mettre les infos de base comme pour Fabmob.
